azure/ConsiderationsForServiceDesign.md

```diff
@@ -141,94 +144,205 @@ cannot collide with a resource path that contains user-specified resource ids.
-which should be represented as one of a fixed set of string values in a field named `status`.
-If the operation is still being processed, the status field will contain a "non-terminal" value, like `Processing`.
+A special case of long-running operation that occurs often is a PUT operation to create a resource
+that involves some additional long-running processing.
+One example is a resource requires physical resources (e.g. servers) to be "provisioned" to make the resource functional.
+In this case, the request may contain an `operation-id` header that the service will use as
+the ID of the status monitor created for the operation.
+
+```text
+PUT /items/FooBar&api-version=2022-05-01
+Operation-Id: 22
+
+{
+   "prop1": 555, 
+   "prop2": "something"
+}
+```
+
+In this case the response to the initial request is a `201 Created` to indicate that the resource has been created.
+The response body contains a representation of the created resource, which is the standard pattern for a create operation.
+A status monitor is created to track the additional processing and the ID of the status monitor
+is returned in the `Operation-Id` header of the response.
+The response may also include an `Operation-Location` header for backward compatibility.
+If the resource supports ETags, the response may contain an `etag` header and possibly an `etag` property in the resource.
+
+```text
+HTTP/1.1 201 Created 
+Operation-Id: 22
+Operation-Location: https://items/operations/22
+etag: "123abc"
+
+{
+  "id": "FooBar", 
+  "etag": "123abc",
+  "prop1": 555,
+  "prop2": "something"
+}
+```
 
-5. After the operation processing completes, a GET request to status monitor URL returns a response with a status field containing a terminal value -- `Succeeded`, `Failed`, or `Canceled` -- that indicates the result of the operation.
-If the status is `Failed`, the status monitor resource must contain an `error` field with a `code` and `message` that describes the failure.
-If the status is `Succeeded`, the response may contain additional fields as appropriate, such as results
-of the operation processing.
+The client will issue a GET to the status monitor to obtain the status of the operation performing the additional processing.
 
-An important distinction between RELO and status monitor LROs is that there is a unique status monitor for each
-status monitor LRO, whereas the status of all RELO operations is combined into the status of the resource.
-So status monitor LROs are "one-to-one" with their operation status, whereas RELO-style LROs are "many-to-one".
+```text
+GET https://items/operations/22?api-version=2022-05-01
+```
+
+When the additional processing completes, the status monitor will indicate if it succeeded or failed.
+
+```text
+HTTP/1.1 200 Ok
+ 
+{
+   "id": "22",
+   "status": "Succeeded"
+}
+```
+
+If the additional processing failed, the service may delete the original resource if it is not usable in this state.
```

Comment:
the "MAY" here might be ambiguous for customer. Seems the only option for them would be always manually check and delete, if service "may" delete the resource.

Reply:
I think the service would need to clearly document this behavior. I suppose we could make that explicit here but I feel that that's generally implied across all topics, so adding it here might mean we have to be explicit everywhere else too.